staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -708,9 +718,10 @@ func (g *genValidations) emitValidationFunction(c *generator.Context, t *types.T
 		"inType":    t,
 		"errorList": c.Universe.Type(errorListType),
 		"fieldPath": c.Universe.Type(fieldPathType),
+		"vCtx":      c.Universe.Type(vCtxType),
 	}
 
-	sw.Do("func $.inType|objectvalidationfn$(obj *$.inType|raw$, fldPath *$.fieldPath|raw$) (errs $.errorList|raw$) {\n", targs)
+	sw.Do("func $.inType|objectvalidationfn$(vCtx $.vCtx|raw$, obj *$.inType|raw$, fldPath *$.fieldPath|raw$) (errs $.errorList|raw$) {\n", targs)
```

Comment:
We will ultimately pass this into the validator plugins, too, right?

Reply:
I think we might. I was holding off until we saw some use cases, but it seem plausible.